distributed/scheduler.py

```diff
@@ -2547,6 +2553,9 @@ def _transition_erred_released(self, key: Key, stimulus_id: str) -> RecsMsgs:
 
         for dts in ts.dependents:
             if dts.state == "erred":
+                # Does this make sense?
+                # This goes via released
+                # dts -> released -> waiting
```

Comment:
Agree this makes no sense to me either. Is there a unit test anywhere to shed light on it?

Reply:
I haven't investigated this any further.